src/components/redditClone/newPost.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react'
+import Form from 'components/core/form'
 
 export default props => {
   return (
     <div>
       <div>
-        <button onClick={() => props.onAddPost('https://www.amazon.co.uk', "Earth's Most Customer- centric Company")}>Create a New Post</button>
+        <Form onSubmit={() => props.onAddPost()}>
+          Link:
```

Comment:
Yea, try again to make this a label, should work. Hit me with the errors if there are some.